### DISCUSSION TITLE
Short press on climate button

### DISCUSSION THREAD

**Discussion by ady1972** [Blackymas/NSPanel_HA_Blueprint#1119] [General]
I'm new of your geat Blueprint and it's really fantastci.
In button page i've assigned climate entities to buttons, but to be able to setup temperature and operating mode i need to long touch on relative button.
Is it possible to do only short touch to do this?

Thanks in advane

**Comment by edwardtfn** [comment]
It is possible, but I'm afraid it won't be consistent with button behavior for other entities domains.
But I think what you are asking for makes sense... maybe is just about find other domains that will require similar behavior (media_player is one on my mind) and standardize those all. Basically, 2-state entities should work short click, for other, short click should be just the same as long click, I think.

**Reply by ady1972** [reply]
thanks a lot for your answer....i'll wait :)

**Comment by Parateam** [comment]
Hello, i have climates on the button Page. Now it is possible to reach the Climate Page on Short Press. 

That's fine, but with long-press there is  the same function. Is it possible the Switch climate on and off? That would be nice. I have only heat and off function on climate.

**Reply by edwardtfn** [reply]
- #1733
